### ISSUE TITLE
Update Screen improvements

### ISSUE THREAD

**Issue by wlhlm** [MultiMC/MultiMC5#1360]
I'm using the MultiMC Development-channel, which means there are occasionally updates, which I install using the built-in update mechanism. The main interface for this is the update window (with changelog, etc.) and it slightly irritates me every time I have to use it. Here are a couple of things:

![updatescreen2](https://cloud.githubusercontent.com/assets/1928113/11888289/27282c7a-a53c-11e5-899e-7fa9ba59347e.png)
- The dialog will always show the same buttons, no matter if there's an update or not. This means when there's no update, you close the dialog by clicking on a button labeled "Don't update yet", which implies that there _is_ an update. Maybe have a "Close" button instead.

![updatescreen1](https://cloud.githubusercontent.com/assets/1928113/11888290/2cbd39f0-a53c-11e5-884f-6855a1a286b6.png)
- this mostly relevant for the dev-channel: currently, the dialog does not show the build number of the new version, so it is unclear to which version you're updating.
 - there is a new build for every commit made to master, but the changelog displayed only shows superficial changes. Maybe include the commit header for each build in the changelog when using the dev-channel

- I find it hard to visually distinguish if there is an update or not. The heading inside the changelog widget is the most prominent feature of the dialog window and it requires some cognitive effort to read the actual heading of the dialog ("No updates found"; "A new update is available!"). Maybe add a color indicator to the window, that immediately suggest if there's an update available or not, maybe in form of a red/green icon in front of the dialog heading.

I realize, that the above issues are personal and every user is different, so I'm very interested in other opinions. I'm no UX designer and the suggestions I made can be taken with a grain of salt.

**Comment by peterix** [comment]
That's a fair observation. I never put any extra work into this and it's meant for the stable builds, really.

I did update the changelog for 0.4.8 a bit before the stable release, but keeping it current is - right now - a manual process. On the other hand, if I used the raw github commits, it wouldn't be as nice.

I'm not sure if I want to touch the changelog until I overhaul the update system again.

Close button should be easy.
